application/django/views.py

```diff
@@ -0,0 +1,11 @@
+from django.http import HttpResponse
+from os import getenv
+
+def index(request):
+    if not (getenv('DEMO_USERNAME') and getenv('DEMO_PASSWORD')):
+        content = 'not all variables are set'
+        status = 500
+    else:
+        content = 'Welcome ' + str(getenv('DEMO_USERNAME'))
+        status = 200
```

Comment:
This feels a little thin. Can we maybe make it more like the example of `secrethub/demo-app`? Since we have the username and password, we could make the same call to the demo API. [Here](https://github.com/secrethub/demo-app/blob/bab6de6a86c586b76d9187b33d6cfe562f1259da/app/serve.go#L58) you can find how that's done in Go (if you don't understand how it works; feel free to ask!).

Reply:
Problem is is that we'll have to do and maintain that for all examples, lots of different languages with often extra 3rd party libraries to include, etc.

Reply:
I agree that it adds overhead. But the downside now is that this example does nothing...  Making a request isn't that complicated, right? Most languages have a really accessible and straightforward way of doing that. And since it's used a lot, it's also something that's not easily broken/deprecated.

Reply:
The goal of this example is to show it's super easy to integrate SecretHub with your application. Therefor I wanted to keep the examples as thin and simple as possible. I wanted to show that you don't need any (special) code in your application. 

With this simple example, it is very clear what is happening. You use secrethub run -e and the secrets are available as environment variables. Therefor I prefer not to use the demo API for this

Reply:
Can someone explain to me what the difference is between an example and a guide in this respect? Wouldn't it be best if we could link to the examples from the guides? If so, they'd have to be similar in functionality.

Reply:
> Wouldn't it be best if we could link to the examples from the guides

Yep, that's the idea.

> If so, they'd have to be similar in functionality.

Exactly, they are similar now (same curl command, status code, response, etc). 

But next to the examples there is the `demo-app` that's embedded in the SecretHub CLI. And the question to answer is: should all the app framework examples in this repo be like the `demo-app`?

Reply:
For me, the answer is: that would be sweet, but not a hard requirement.